packages/magento-product-configurable/components/ConfigurablePrice/ConfigurablePriceTiers.tsx

```diff
@@ -0,0 +1,24 @@
+import {
+  ProductPagePriceTiers
+} from '@graphcommerce/magento-product/components/ProductPagePrice/ProductPagePriceTiers';
+import { ConfigurableOptionsFragment } from '../../graphql/ConfigurableOptions.gql'
+import { useConfigurableOptionsSelection } from '../../hooks/useConfigurableOptionsSelection'
+
+type ConfigurablePriceProps = {
+  product: ConfigurableOptionsFragment
+  index?: number
+}
```

Comment:
Might be an idea to also forward the props to ProductPagePriceTiers.

Reply:
Not sure if I understand this correctly. What would I do with those props in `ProductPagePriceTiers`?